tests/pytests/functional/pillar/test_gpg.py

```diff
@@ -184,14 +184,34 @@
     },
 }
 
+# Random data pretending to be ciphertext
+GPG_PILLAR_YAML_FAIL = """\
+fail: |
+  -----BEGIN PGP MESSAGE-----
+
+  OzyJmQJJVPxGQqyxwIcAl0wWqdSTHpMXYPLrDoRU8H1xa2DhE5DeUihjm4fHUcHp
+  -----END PGP MESSAGE-----
+"""
+
+GPG_PILLAR_ENCRYPTED_FAIL = {
+    "fail": (
+        "-----BEGIN PGP MESSAGE-----\n"
+        "\n"
+        "OzyJmQJJVPxGQqyxwIcAl0wWqdSTHpMXYPLrDoRU8H1xa2DhE5DeUihjm4fHUcHp\n"
+        "-----END PGP MESSAGE-----\n"
+    ),
+}
+
```

Comment:
Would you mind turning these (and the other globals in this file) into pytest fixtures?